tests/autorestart/test_container_autorestart.py

```diff
@@ -418,6 +419,12 @@ def run_test_on_single_container(duthost, container_name, service_name, tbinfo):
         logger.info("Restore auto-restart state of container '{}' to 'disabled'".format(container_name))
         duthost.shell("sudo config feature autorestart {} disabled".format(feature_name))
 
+    if container_name == "syncd":
+        # During syncd TC bgp service reach start-limit-hit, but at the time when it is
+        # checked in is_hiting_start_limit bgp service have another state.
+        # Wait 100 seconds to stabilize bgp state
+        time.sleep(100)
```

Comment:
It's direct and effective. But can we make it general? In this test script, after each feature container was tested, it will post-check all containers' health in `check_all_critical_processes_status`, and this operation was executed in `wait_util`, so that can we put the `start-limit-hit` check here? I think it may have two advantages.
1. It will ignore whatever container testing triggered bgp or other services into start-limit-hit.
2. `wati_util` has a parameter `timeout`, it was controlled by static variable `POST_CHECK_THRESHOLD_SECS` and default is `360`s, it's enough to make bgp or other service state stable.